README.md

```diff
@@ -3,35 +3,93 @@
 </br>
 </br>
 
-<div align="center">
+<p align="center">
+  <img src="assets/logo.png" alt="xk6-disruptor" width="300"></a>
+  <br>
+  "Like Unit Testing, but for <strong>Reliability</strong>"
+  <br>
+</p>
+<p align="center">  
+</div>
 
-![logo](assets/logo.png)
+<blockquote align="center">
+⚠️ <strong>Important</strong> ⚠️ 
+<br>This project is still in its early stages and under active development.
+<br>The API is subject to change, and there may be bugs.
+<br>Use at your own risk. Thanks!
+</blockquote>
```

Comment:
I don't like how this note looks centered below the logo.  I would move below the next paragraph

```suggestion
```

Reply:
I don't find how to add this block after the next paragraph because there is already another suggestion on that paragraph. I think you can add this block later in another commit:

Reply:
Let me try :+1: